### ISSUE TITLE
Alliance succubus quest

### ISSUE THREAD

**Issue by Kirishiki** [dalaranwow/dalaran-wow#5119]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
As human warlock you need to travel ironforge to get quest "galkin summons" that sends u to stormwind to get succubus quest,,
**Expected behaviour**: Tell us what should happen instead
i think humans shouldnt need this send quest at all but get succubus quest right away in stormwind .
**Steps to reproduce the problem**:

1. change so humans shouldnt need to do the galkin summons quest (i havent done any research if this is retail like but it wud seem more understandable)
2. 
3. 

**Include proofs for this behaviour**
i tryed finding the succubus quest today from goldshire and stormwind but couldnt find it so went to ironforge and there was galkin summons..
You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Hello and thank you for the report @Kirishiki.
Here is the alliance version of the quest:
https://wowgaming.altervista.org/aowow/?quest=1717
I can't seem to find any issues with it.

Will reopen if there is aditional information.
